### DISCUSSION TITLE
klay thompson選擇錢還是情誼

### DISCUSSION THREAD

**Discussion by Ryanlolry** [1101-GS4719-C/Python-Community-GS4719-C#1646] [General]
klay thompson在 2019年冠軍賽時受傷後，球隊基於他前期的貢獻，仍給出一份超出預期的頂新合約，但他實力卻大不如前。近期傳出他想要一份高價合約，甚至考慮去其他隊來獲得高價合約。
我認為基於他已經在這個球隊待了近十年，而這十年也幫助球隊或的四座冠軍，我希望他可以續留原球隊，並以較低且長期約續留球隊，這樣做不但能繼續與熟悉隊友配合，也能獲得一人一城的美譽，也希望他能替球隊考慮，畢竟在他受傷時，球隊也給他足夠的尊重。但他想要獲得更大的合約也情有可原，畢竟這會是他生涯最後一份大約。
資料參考:
https://tw.sports.yahoo.com/news/%E6%B5%AA%E8%8A%B1%E5%85%84%E5%BC%9F%E6%9C%83%E6%8B%86%E5%A4%A5-%E6%B2%83%E7%A5%9E%E7%88%86%E6%96%99-k%E6%B9%AF%E5%92%8C%E5%8B%87%E5%A3%AB%E7%BA%8C%E7%B4%84%E6%B2%92%E4%BB%BB%E4%BD%95%E9%80%B2%E5%B1%95-040931589.html

**Comment by 12Jesse24** [comment]
個人覺得均薪2500不能再多，就看K湯願不願意接受，況且勇士之前也給過高薪，K湯自己應該要能察覺降薪留隊應該是目前最好的選擇!

**Comment by Tomken123** [comment]
k湯真的回不了，心氣還可以這麼高，剩舉四根手指的功能，我想新竹工程師再過不久就要傳訊息邀請他了了

**Comment by Ryanlolry** [comment]
我也這麼認為，他要先意識到自己已經比以前爛了，適當的改變自己的定位，延長自己的生涯長度，畢竟感覺他不要亂搞，永世都至少會給他一份溢價一點點的合約

**Comment by KH1126** [comment]
作為一個支持Thompson七年以上球迷，
我認為留在勇士對雙方都有幫助，
畢竟以Curry巨大防守吸引力的情況下，
隊上有一個38%以上3分命中率選手，可謂是幫助巨大。

**Comment by SusuTima** [comment]
我是Curry的忠實粉絲（從14年到現在），其實Curry四冠已經成就滿滿了，接下來只希望Curry可以不受傷慢慢的打到退休。至於Thompson，如果能看到三老都終老勇士那當然是最好，但如果退休前想再撈一筆，當然也是情有可原。反正不論結果如何，只要大家都健健康康的就好。

**Comment by wanderer657** [comment]
Thompson和勇士合約問題牽扯到球隊未來和團隊氛圍，他的離開可能影響戰術體系和球隊聲譽。儘管他在傷病中表現出色，但年齡和身體狀況成為考量。保留Thompson對於勇士隊穩定競爭力至關重要，且他的離開可能對球迷忠誠度和商業價值造成負面影響。因此，勇士需謹慎考慮平衡短期和長期利益。
